ui/packages/ui/src/app/pages/connectors/ConnectorListItem.tsx

```diff
@@ -46,7 +47,7 @@ export const ConnectorListItem: React.FunctionComponent<IConnectorListItemProps>
 
   const getTaskStateStr = (taskStates: any) => {
     const stateMap = new Map(Object.entries(taskStates));
-    return Array.from(stateMap.values()).join(", ");
+    return Array.from(stateMap.values());
   }
 
```

Comment:
I think this function can just return the Map now - and use the map keys for the task display